pkg/option/runtime_options.go

```diff
@@ -7,7 +7,7 @@ const (
 	PolicyTracing        = "PolicyTracing"
 	ConntrackAccounting  = "ConntrackAccounting"
 	ConntrackLocal       = "ConntrackLocal"
-	Debug                = "Debug"
+	DatapathDebug        = "DatapathDebug"
```

Comment:
I don't think this const should be renamed, this is the name of the global debug option also infuencing log level, cf. https://github.com/cilium/cilium/blob/1a0553ccaf8ac26a05add34355ae931dd71f4158/daemon/cmd/daemon.go#L1310-L1312

Reply:
Shall I revert that back?

Reply:
Yes, please only rename the option when concerning the datapath debug flag. The global daemon debug flag should remain as is.